src/functions.cpp

```diff
@@ -1,10 +1,10 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include "functions.hpp"    
 #include <algorithm>
 #include <iostream>
 #include <limits>
 
 void print(std::vector<int> const & numbers) {
-    for (int i = 0; i < numbers.size(); ++i) {
+    for (int i = 0; i < int(numbers.size()); ++i) {
```

Comment:
I think it is not the best solution. It would be better to replace `int i 1 = 0` with `std::size_t i = 0`

Reply:
yes, please fix it as @Kasperice suggested.